action.yml

```
@@ -88,6 +88,10 @@ inputs:
     description: 'Only issues or pull requests with all of these labels are checked if stale. Defaults to `[]` (disabled) and can be a comma-separated list of labels.'
     default: ''
     required: false
+  any-of-labels:
+    description: 'Only issues or pull requests with one of these labels are checked if stale. Defaults to `[]` (disabled) and can be a comma-separated list of labels.'
```

```suggestion
    description: 'Only issues or pull requests with one of these labels are checked if stale. Defaults to `` (disabled) and can be a comma-separated list of labels.'
```
It seems that this "lie" about the default value is coming from the current code base and this suggestion can also be made for the existing `only-labels` option.

---

> It seems that this "lie" about the default value is coming from the current code base and this suggestion can also be made for the existing `only-labels` option.

Lol yep, that did seem strange... I'll fix it in both of them 😆